site/content/contribute/desktop/testing.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Automated Testing"
+heading: "Testing the Desktop App"
+description: "A guide to writing automated tests for the Desktop App"
+date: 2019-01-22T00:00:00-05:00
+weight: 4
+---
+
+For most changes to the Desktop App, we should consider writing an automated test to ensure that the change or fix is maintained in the codebase. 
+Depending on the nature of the change, we can write a unit test or an E2E (end-to-end, or integration) test.
+
+## Unit Tests
+
+We use the `jest` test runner to run unit tests in the Desktop App.
+You can run the following command to run the tests:
+
+    npm run test:unit
+
+Unit tests are usually written for parts of the `common` and `main` modules, and usually cover individual functions or classes.
```

Comment:
An example of this would be great. Can we link to something on GitHub that demonstrates an ideal unit test.